staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/validating/plugin.go

```diff
@@ -108,6 +119,25 @@ func NewPlugin(_ io.Reader) *Plugin {
 					NewValidatingAdmissionPolicyBindingAccessor,
 					compilePolicy,
 					f,
+					func(hooks []generic.PolicyHook[*Policy, *PolicyBinding, PolicyEvaluator]) {
+						var allowCount, warnCount, denyCount int
+						policyCount := len(hooks)
+						for _, hook := range hooks {
+							for _, binding := range hook.Bindings {
+								action := getValidatingAction(binding.Spec.ValidationActions)
```

Comment:
e.g. if `validationActions: [Warn, Audit]` is specified, only `Warn` will be counted?  Only `validationActions: [Audit]` would allowCount++ ?

Reply:
That was the idea, just to track whether the request was denied, warned or allowed.  I was treating the audit part as orthogonal for the purpose of the outcome.

Reply:
Note: currently ["Warn","Audit"] would increment the warn count and not the allow count

Reply:
I just feel the way divid binding now especially the `allow` label is confusing -_-